src/Resources/views/form_layout.html.twig

```diff
@@ -27,7 +27,7 @@
 {% block attributes %}
     {% if type is defined %}
         {%  if type is same as('datetime-local') or type is same as('date') or type is same as('time') %}
-            {{ stimulus_controller('whatwedo/core-bundle/datetime') }}
+            {{ stimulus_controller('whatwedo/core-bundle/datetime', {lang: 'de-DE'}) }}
```

Comment:
@marcwieland95 @tuxes3  we should obtain the language from the backend

Reply:
You're absolutely right. Forgot about that actually.